src/Microsoft.OData.ModelBuilder/Vocabularies/VocabularyTermConfigurationShortcutsExtensions.cs

```diff
@@ -19,5 +21,149 @@ public static class VocabularyTermConfigurationShortcutsExtensions
         /// <returns></returns>
         public static PermissionTypeConfiguration HasScopes(this PermissionTypeConfiguration permissionTypeConfiguration, params string[] scopeNames)
             => permissionTypeConfiguration?.HasScopes(scopeNames.Select(scope => new ScopeTypeConfiguration().HasScope(scope)).ToArray());
+
+        /// <summary>
+        /// List of revisions
+        /// </summary>
+        /// <param name="navigationSourceConfiguration"></param>
+        /// <param name="description"></param>
+        /// <param name="version"></param>
+        /// <param name="date"></param>
+        /// <param name="removalDate"></param>
+        /// <returns></returns>
+        public static RevisionsConfiguration HasDeprecation<T>(
+            this NavigationSourceConfiguration<T> navigationSourceConfiguration,
+            string description,
+            string version,
+            DateTime date,
+            DateTime removalDate)
+            where T : class
+        {
+            return navigationSourceConfiguration?
+                .HasRevisions()
+                .HasRevisions(a => a
```

Comment:
It doesn't make sense to call twice for "HasRevision()" ?

Reply:
solved